specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-12-01-preview/managedcluster.json

```diff
@@ -992,10 +992,12 @@
       "properties": {
         "healthCheckWaitDuration": {
           "type": "string",
+          "x-ms-format": "duration-constant",
```

Comment:
>"x-ms-format": "duration-constant" [](http://example.com/codeflow?start=10&length=34)

Is this change due to a backend change in validation on these properties, or is it to make the spec match the existing backend? If it's the former, it requires breaking change review and might need a new API version. If it's the latter, then it should be fine.

Reply:
it's the latter